frontend/Dockerfile

```diff
@@ -64,6 +64,7 @@ WORKDIR /frontend
 
 # Update system and install security updates
 RUN apt-get update \
+    && apt-get upgrade -y --no-install-recommends \
```

Comment:
@coilysiren I was hitting some more Trivy scan issues so had to add an update command here

Reply:
👍